docs/eventing/sources/sinkbinding/getting-started.md

```diff
@@ -0,0 +1,300 @@
+---
+title: "Create a SinkBinding object"
+weight: 02
+type: "docs"
+---
+
+![API version v1](https://img.shields.io/badge/API_Version-v1-red?style=flat-square)
+
+This topic describes how to create a SinkBinding object and connect it to a
+subject in your cluster.
+
+If you follow the examples in this topic, you will have a SinkBinding object
+that resolves a Knative Service (the sink) into a URI, and sets that URI in the
+environment variable `K_SINK` in new jobs from a cron job (the subject).
+The SinkBinding updates `K_SINK` if the URI changes.
+
+If you have an existing subject and sink, you can replace the examples with your
+own values.
+
+## Before you begin
+
+Before you can create a SinkBinding object, you must:
+
+- Have Knative Serving installed on your cluster.
+- Optional: If you want to use `kn` commands with SinkBinding, install the `kn` CLI.
+
+
+## Optional: Choose SinkBinding namespace selection behavior
+
+The SinkBinding object operates in one of two modes: `exclusion` or `inclusion`.
+
+The default mode is `exclusion`.
+In exclusion mode, SinkBinding behavior is enabled for the namespace by default.
+To disallow a namespace from being evaluated for mutation you must exclude it
+using the label `bindings.knative.dev/exclude: true`.
+
+In inclusion mode, SinkBinding behavior is not enabled for the namespace.
+Before a namespace can be evaluated for mutation you must
+explicitly include it using the label `bindings.knative.dev/include: true`.
+
+### Set to inclusion mode
+
+To set to inclusion mode, edit the `eventing-webhook` deployment to set
+`SINK_BINDING_SELECTION_MODE` to `inclusion`.
+The mode determines the default scope of the webhook.
+
```

Comment:
Can there be a better example here showing where / how this is actually set as a label, or a command for it?